README.md

```diff
@@ -37,9 +45,9 @@ ansible-playbook deploy-all.yml
 
 **Note:** All the WebUIs are Kerberized, you need to have a working Kerberos client on your host, configure the KDC in your `/etc/krb5.conf` file and obtain a valid ticket. You can also access the WebUIs through [Knox](#knox).
 
-## Customised deployment
+## Customised Deployment
 
-Each of the below sections includes a high-level explanation of each possible step of the deployment using this repository.
+Each of the below sections includes a high-level explanation of each possible step of TDP deployment.
```

Comment:
Each of the below sections includes a high-level explanation of each possible step of a TDP deployment.

Reply:
Yes but I think the commit hash will change upon merge as the PR is squashed. So we have to wait until https://github.com/TOSIT-IO/tdp-collection-extras/pull/16 is merged.